wgsl/index.bs

```diff
@@ -41,11 +41,32 @@ thead {
 
 ## Goals ## {#goals}
 
- * Trivially convertable to SPIR-V
- * Constructs are defined as normative references to their SPIR-V counterparts
- * All features in [SHORTNAME] are directly translatable to SPIR-V. (No polymorphism, no general pointers, no overloads, etc)
- * Features and semantics are exactly the ones of SPIR-V
- * Each item in this spec *must* provide the mapping to SPIR-V for the construct
+In no particular order:
+
+ * Text-based
+ * Readable and writable by programmers
+ * Mathematially [one-to-one](https://en.wikipedia.org/wiki/Injective_function) with SPIR-V
+ * Supporting simple and efficient compilation to and from SPIR-V
+ * Supporting simple and efficient compilation to [Metal Shading Language](https://developer.apple.com/metal/Metal-Shading-Language-Specification.pdf) and [HLSL](https://docs.microsoft.com/en-us/windows/win32/direct3dhlsl/dx-graphics-hlsl)
+ * Directly represents structured control flow
+ * Strongly statically typed
+ * As small as possible to represent what we need
```

Comment:
Probably better to just to say "as small as possible." The other words aren't really necessary.